src/client.ts

```diff
@@ -594,13 +569,13 @@ export class CustomTypesClient {
 			// Forbidden
 			// - Missing token
 			// - Incorrect token
-			// - Hard limit reached for bulk request (cannot process)
+			// - Has existing documents (cannot process)
 			case 403: {
 				const json = await res.json();
 
-				if ("details" in json) {
-					throw new BulkTransactionLimitError(
-						"The bulk transaction reached or surpassed the limit of allowed commands.",
+				if ("hasExistingDocuments" in json && json.hasExistingDocuments) {
+					throw new BulkUpdateHasExistingDocumentsError(
+						"The bulk update changes contain deletion operations for custom types with existing documents.",
```

Comment:
💡 #idea: This phrasing gives the reader some direction:

> A custom type with published documents cannot be deleted. Delete all of the custom type's documents or remove the delete operation from the request before trying again.

If it's possible to know which custom type blocks the operation, maybe we could provide it in the message and as an instance property on the error (e.g. `customTypeID`):

> The "${customTypeID}" custom type has published documents and cannot be deleted. Unpublish all of its documents or remove the delete operation from the request before trying again.

Reply:
No, we don't know from the custom type API. That's why later on we are fetching the list of docs per custom types we delete :/
Maybe it should be something to improve on customtypes-api side, but not yet done

Reply:
In that case, would you be good with updating the error message to something like this?

> A custom type with published documents cannot be deleted. Delete all of the custom type's documents or remove the delete operation from the request before trying again.

Reply:
Done